as/plugins/org.jboss.ide.eclipse.as.core/plugin.properties

```diff
@@ -110,6 +110,10 @@ jboss.eap.version.72.name=Red Hat JBoss Enterprise Application Platform 7.2
 jboss.eap.version.72.description=Red Hat JBoss Enterprise Application Platform (EAP) 7.2
 jboss.eap.version.72.runtime.name=Red Hat JBoss Enterprise Application Platform 7.2 Runtime
 
+jboss.eap.version.73.name=Red Hat JBoss Enterprise Application Platform 7.3
```

Comment:
Shouldn't we add "Tech preview" wording so that potential users (those installing it from nightly update site) using it for eap 7.3 beta testing will be aware of the level of quality of this feature?

Reply:
@odockal imho this depends on the goal and imho needs clarification:
* release when eap 7.3 is officially released -> no 
* release as preview so that eap 7.3 Beta can be tested -> yes
Which one is the aim here is not clear to me and we should ask PM to specify.

Reply:
@adietish There is also a time factor in play. They (EAP) needs to provide some tooling that supports beta release (at least in nightly builds). Then, there will be 7.3 server adapter ready in 12.13.0.GA release which will be prepared for future release of 7.3 EAPm right, but that will be GA already, right? So possible preview wording will be at play for only a month, or so, right?

I am just assuming and discussing based on [this]. Otherwise I agree, we need clarification, but when then? Tomorrow cabal? I have a feeling that we need to have this done this week, so thursday's 5x5 might be a bit late...

[this]: https://issues.jboss.org/browse/EAP7-1128

Reply:
ok, sounds like they explicitly want support for Betas, I can add the "Tech Preview" label, no big deal.

Reply:
@adietish Probably. We used the wording "(Tech Preview)" in CDK server adapter [1].
And yes, this might be needed on all places even for runtime. What I see now, seems perfect!

[1]:https://github.com/jbosstools/jbosstools-openshift/commit/de7d0c56d14d1fbbbf51c8b9ce4d1bb89ce422e5#diff-ea109c8904b56a26b9f9d2be65af632cR21

Reply:
@adietish Thanks for changes, I am going to test it.